agent/custom/support_action.py

```
@@ -0,0 +1,687 @@
+
+    # ---------- 主动技能匹配(单帧) ----------
+    def _check_active_skills(self, img, bx, by, actives):
+        for i, v in enumerate(actives):
+            if i >= len(SKILL_ACTIVE):
+                break
+            if not self._match_skill(img, bx, by, SKILL_ACTIVE[i], v):
+                return False
+        return True
+
+    # ---------- 视图判断(主动/被动) ----------
+    def _match_view(self, img, bx, by, skill_dir):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_match_view`

Add return type annotation: `str | None`

(ANN202)